wg-reliability/README.md

```diff
@@ -0,0 +1,36 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Reliability Working Group
+
+Allow users to safely use Kubernetes for managing production workloads by ensuring Kubernetes is stable and reliable.
```

Comment:
This one sentence feels very broad for scope.  Scope was a large part of the other discussion threads before this PR. Without asking you to go all the way into including a draft charter doc in this PR, could some of that discussion be distilled into a few more sentences or bullet points here tentatively describing the expected outline shape of "ensuring Kubernetes is stable and reliable"?

Reply:
ie: this info exists, it'll just be hard to find for a reader of the updated sigs list file once this merges and they're jumping off to this PR's current readme unless they look at the file git history and find this pr and the links in its description and go to them.

Reply:
@tpepper this is the README, this WG still has to file a charter with the scope ( example https://github.com/kubernetes/community/blob/master/wg-k8s-infra/charter.md#scope )

Reply:
which will be roughly what is here:
https://docs.google.com/document/d/1IZnIG26FOJXWvW5n0Yu6zL89pH4cO8NYHKMiESc2fV0/edit#

I'm not entirely sure it makes sense to put it here - if the WG will get approved I can put together a charter PR in the next day (basically migrating this doc to the markdown in this repo).

Reply:
Are we expecting a fast chartering here?  Some WG's have been quite slow.  If this one gets slow, at least a minimal summary of potential scope vision and a link to the draft charter would be beneficial in the meantime for somebody arriving at the sig-list.md without connection to details.

Reply:
As soon as this is approved, I will open PR with the charter (mostly what is in https://docs.google.com/document/d/1IZnIG26FOJXWvW5n0Yu6zL89pH4cO8NYHKMiESc2fV0/edit#heading=h.jrj8xt906vjp
). Given more-or-less agreement on that (number of people already looked at this and commented), I think it hopefully shouldn't be very controversial, so hopefully it should come almost immediately after WG approval.